lib/show.rb

```diff
@@ -0,0 +1,14 @@
+class Show
+  attr_reader :name, :creator, :characters, :total_salary
```

Comment:
you don't need an attr_reader for total_salary if you have a method of the same name